Doc/library/functions.rst

```diff
@@ -982,6 +978,10 @@ are always available.  They are listed here in alphabetical order.
       See the :ref:`integer string conversion length limitation
       <int_max_str_digits>` documentation.
 
+   .. deprecated-removed:: 3.11 3.14
+      :class:`int` will no longer delegate to the :meth:`~object.__trunc__` method.
```

Comment:
Perhaps ``` :func:`int` ``` would be better for this. `int` is documented as a function and a class. In this case we refer to a callable.